src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -215,6 +216,11 @@ require.config({
     'draw.io': {
       deps: ['mxgraph-editor', 'base64', 'pako-global', 'spin-global', 'jszip']
     }
+  },
+  map : {
+    'diagram-utils' : {
+      'mxgraph-common' : 'mxgraph-editor'
```

Comment:
```suggestion
      'mxgraph-common': 'mxgraph-editor'
```
I see you are used to leave a space before ``:``. So far our code style is to leave a space only after ``:`` (same as with ``,``).